intel_extension_for_transformers/neural_chat/docker/inference/README.md

```diff
@@ -15,12 +15,11 @@ Option 2: If you need to use proxy, please use the following command.
 docker build --network=host --tag chatbotinfer:latest  ./ --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${http_proxy} -f Dockerfile  --target cpu  
 ```
 
+Then mount the model files to the docker container using '-v'. Make sure using the absolute path for host_dir.
```

Comment:
any limitation for host_dir/mount_dir?

Reply:
'host_dir' should be absolute path, 'mount_dir' is the path users specify in the docker, which is the same as the path when run inference later.